autoprotocol/container_type.py

```diff
@@ -1197,6 +1197,41 @@ def row_count(self):
     cat_no="353077",
 )
 
+#:
+FLAT96DELTA = ContainerType(
+    name="Nunc MicroWell 96-Well, Nunclon Delta-Treated, Flat-Bottom Microplate",
+    shortname="96-flat-white-dc",
+    is_tube=False,
+    well_coating=None,
```

Comment:
is the Nunclon Delta a well coating treatment?

Reply:
It's an "energy-treated surface".  Leaving `None` for now until standardization of "well_coating" properties.